JustSaying.IntegrationTests/JustSayingFluently/MultiRegion/WithSqsPointToPointSubscriber/WhenAFailoverRegionIsSetup.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace JustSaying.IntegrationTests.JustSayingFluently.MultiRegion.WithSqsPointToPointSubscriber
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable", Justification = "Acceptable in tests")]

```

Comment:
If the blanket statement in the tests is to just turn this off, it would be less effort to just add a `<NoWarn>` for `CA1001` to the `csproj` file.

Reply:
Good point, will do

Reply:
It is done